test/e2e/specs/create-voucher-flow.e2e.ts

```diff
@@ -0,0 +1,103 @@
+class CreatePage {
+  get createButton() {
+    return $(
+      "body > div.top_page_container > div.NumPad_numpad__yb7Im > div > button:nth-child(5)"
+    );
+  }
+
+  get submitButton() {
+    return $(
+      "body > div.top_page_container > div.CreateLink_commission_and_submit_buttons__ydXyN > button.Button_EnabledButton__uINCR.undefined"
+    );
+  }
+
+  async clickCreateButton() {
+    await (await this.createButton).waitForDisplayed();
```

Comment:
avoid having 2 await on the same line. create a property first in the first line and then reuse in the second line.